src/pages/RepoPage/CommitsTab/CommitsTableTeam/CommitsTableTeam.spec.tsx

```diff
@@ -0,0 +1,255 @@
+    username: 'codecov-user',
+    avatarUrl: 'cool-avatar-url',
+  },
+  compareWithParent: {
+    __typename: 'Comparison',
+    patchTotals: {
+      percentCovered: 90,
+    },
+  },
+}
+
+const node3 = {
+  ciPassed: false,
+  message: 'commit message 3',
+  commitid: '7822fd88f36efcd9af276792813a83da17bd3c67',
+  createdAt: '2023-10-13T00:00.000000',
+  author: {
+    username: 'codecov-user',
+    avatarUrl: 'cool-avatar-url',
+  },
+  compareWithParent: {
+    __typename: 'Comparison',
+    patchTotals: {
+      percentCovered: 100,
+    },
+  },
+}
+
+const server = setupServer()
+const wrapper =
+  (queryClient: QueryClient): React.FC<React.PropsWithChildren> =>
+  ({ children }) =>
+    (
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter initialEntries={['/gh/codecov/cool-repo/commits']}>
+          <Route path="/:provider/:owner/:repo/commits">{children}</Route>
+        </MemoryRouter>
+      </QueryClientProvider>
+    )
+
+beforeAll(() => {
+  server.listen()
+})
+
+afterEach(() => {
+  server.resetHandlers()
+})
+
+afterAll(() => {
+  server.close()
+})
+
+interface SetupArgs {
+  noEntries?: boolean
+}
+
+describe('CommitsTableTeam', () => {
+  function setup({ noEntries = false }: SetupArgs) {
+    const queryClient = new QueryClient()
+
+    server.use(
+      graphql.query('GetCommitsTeam', (req, res, ctx) => {
+        if (noEntries) {
+          return res(
+            ctx.status(200),
+            ctx.data({
+              owner: {
+                repository: {
+                  __typename: 'Repository',
+                  commits: {
+                    edges: [],
+                    pageInfo: {
+                      hasNextPage: false,
+                      endCursor: null,
+                    },
+                  },
+                },
+              },
+            })
+          )
+        }
+
+        const dataReturned = {
+          owner: {
+            repository: {
+              __typename: 'Repository',
+              commits: {
+                edges: req.variables.after
+                  ? [
+                      {
+                        node: node3,
+                      },
+                    ]
+                  : [
+                      {
+                        node: node1,
+                      },
+                      {
+                        node: node2,
+                      },
+                    ],
+                pageInfo: {
+                  hasNextPage: req.variables.after ? false : true,
+                  endCursor: req.variables.after
+                    ? 'aa'
+                    : 'MjAyMC0wOC0xMSAxNzozMDowMiswMDowMHwxMDA=',
+                },
+              },
+            },
+          },
+        }
+        return res(ctx.status(200), ctx.data(dataReturned))
+      })
+    )
+
+    return { queryClient }
+  }
+
+  describe('renders table headers', () => {
+    it('renders name column', async () => {
+      const { queryClient } = setup({})
```

Comment:
Consider moving these 4 lines to their own function.

Reply:
When it comes to testing our philosophy is that these test setups should be unique to each test, that way if you need to change something on one test there's less overall work you have to do.